.gitignore

```diff
@@ -328,3 +328,5 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+/src/PeopleLookup.Mvc/Properties/serviceDependencies.*

```

Comment:
never seen anything like this before -- is it generated by some tool?

Reply:
@srkirkland I think the latest VS 2019 does it. It is json stuff. I think @sprucely encountered it in sloth too.